### PR TITLE
Fixes Accept header to allow legacy and laminas-api-tools

### PR DESCRIPTION
**Pull request by michalbundyra**
|    Q          |   A
|-------------- | ------
| Bugfix        | yes/no

### Description

On migration accept/content-type header has been changed to invalid value. Now we accepts both:
- new: `application/vnd.laminas-api-tools.v1.config+json`
- legacy: `application/vnd.zfcampus.v1.config+json`

### CODE REVIEW

**Comment by michalbundyra** [comment]
@weierophinney We might want apply this patch to all tags. This is migration issue. Not sure what we should do with `application/vnd.laminascampus.v1.config+json`. I think we might want to keep it as well, as some people might already migrate and use that invalid value.

Thoughts?

**Comment by froschdesign** [comment]
@michalbundyra 
> I think we might want to keep it as well, as some people might already migrate and use that invalid value.

I had the same thought. I'm for keeping the wrong name and open an issue report to remove it with the next major version.

File: src/Controller/AbstractConfigController.php [hunk 1]
```diff
@@ -114,7 +114,8 @@ protected function getHeaderType($headers, $header)
         $accept = strtolower(trim($accept));
         switch ($accept) {
             case 'application/json':
-            case 'application/vnd.laminascampus.v1.config+json':
```
Comment (weierophinney) [inline src/Controller/AbstractConfigController.php:117]: Agreed with @froschdesign here - we should keep this, and add a `@todo` to remove it for next major.

Comment (michalbundyra) [reply src/Controller/AbstractConfigController.php:117]: Added back, so now we have three options, but in next major we want to keep only one: `application/vnd.laminas-api-tools.v1.config+json`.

**Pull request merged**